solrb.gemspec

```diff
@@ -1,16 +1,15 @@
-
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "solrb/version"
+require "solr/version"
 
 Gem::Specification.new do |spec|
   spec.name          = "solrb"
-  spec.version       = Solrb::VERSION
-  spec.authors       = ["Adriano Luz"]
-  spec.email         = ["adriano.luz@machinio.com"]
+  spec.version       = Solr::VERSION
+  spec.authors       = ["Adriano Luz", "Valentin Vasilyev"]
+  spec.email         = ["adriano.luz@machinio.com", "valentin@machinio.com"]
 
-  spec.summary       = "Solr Ruby client"
-  spec.homepage      = "https://github.com/machinio/solrb"
+  spec.summary       = "Solr Ruby client with a nice object-oriented API"
+  spec.homepage      = "https://github.com/machinio/solr"
```

Comment:
Will we change the repository name as well?

Reply:
oh, it's by accident I used grep + sed to replace all occurrences, will fix